_data/simple-icons.json

```diff
@@ -14374,6 +14374,11 @@
             "source": "https://www.sncf.com",
             "guidelines": "https://www.sncf.com/fr/groupe/marques/sncf/identite"
         },
+        {
+            "title": "Snort",
+            "hex": "F6A7AA",
+            "source": "https://snort.org/"
```

Comment:
In your PR description you said "Svg source from : [icon-icons.com/fr/icone/Snort/131984](https://icon-icons.com/fr/icone/Snort/131984)". Please use the source from the website if there is an available SVG!

Reply:
There is an available svg, but only colorized. Do you have a tool to make it black and white ?
Here is the colorized svg from the official website : https://www.snort.org/assets/logo_snort_color.svg

Reply:
Ah perfect, that works. Most logos are colorized! You could/should use [Inkscape](https://inkscape.org/) here though any half-decent SVG editor should support changing colors and merging paths :sweat_smile:

Reply:
I was able to get this using the svg you sent previously
![image](https://github.com/simple-icons/simple-icons/assets/46492476/43131cdb-8f69-4c10-aebd-959c035d35b5)
Does it look better to you? If yes, I will modify my PR to use this one

Reply:
That image looks blank to me.

Reply:
Sorry, a bug with the copy on preview, fixed now

Reply:
Looks much better! I think that using the SVG from that link you provided earlier though might be better. Where did you find that SVG from?

Reply:
This one come from the one you extracted from their icon font used on their website
I don't know how you were able to extract it, but I was able to clean it (using the same tools as previously, https://boxy-svg.com/app and SVGO)

Reply:
Here is the original SVG you extracted : 
```
<?xml version="1.0" standalone="no"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd" >
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" viewBox="-10 0 582 512">
   <path fill="currentColor"
d="M183 313c-3 -7 8 -16 13 -13c-3 4 -7 12 -13 13zM454 273c13 -16 42 -47 72 -58c4 5 -12 10 -13 11c-25 44 -32 48 -51 74c-36 48 -62 53 -67 77c-1 -28 35 -73 59 -104zM556 157c15 24 -7 64 -18 84c-41 75 -89 127 -157 168c-10 6 -23 12 -40 15c-11 1 -21 1 -30 -2
c-10 -5 -21 -14 -27 -24c-12 -21 -15 -43 -28 -65c-4 -7 -13 -16 -13 -16l-11 17s14 9 9 20c-3 7 -12 12 -20 10c-7 -1 -15 -10 -15 -10s-20 8 -30 10c26 14 14 38 14 38s4 5 7 8c4 5 8 14 7 20c-2 32 -29 47 -52 48c26 -10 34 -37 18 -54c-25 6 -54 0 -68 -17
c-10 -12 2 -25 17 -30c10 -3 31 0 31 0s-8 -9 -11 -10c-30 -1 -48 0 -74 -15c-12 7 -24 15 -41 12c-9 -1 -17 -6 -13 -19c2 -10 11 -17 19 -25c-30 -22 -13 -88 20 -120c-17 -4 -26 -11 -28 -18c-3 -12 7 -23 16 -27c14 -6 29 -3 42 0c12 -23 31 -41 49 -60
c-1 -18 -9 -42 -9 -63c12 1 27 1 39 1c7 0 21 -13 37 -23c6 12 18 39 18 39s1 0 3 -1c45 -26 102 -18 119 27c9 -6 23 -12 32 -19c9 14 16 37 22 49c-11 2 -24 4 -34 -1c-6 -1 -8 1 -12 4c3 4 -2 16 -1 16c23 13 105 33 167 19c17 -3 38 2 46 14zM363 67c0 0 -15 7 -23 11
c10 10 3 15 6 24c6 -5 8 -11 8 -11l12 8l12 2s-6 -15 -9 -22c-2 -5 -6 -12 -6 -12zM196 430c-3 -15 -22 -38 -38 -44c-30 -12 -45 2 -46 10c-3 19 37 24 38 23c0 0 14 0 7 -10c-12 -2 -26 -9 -26 -9c4 -5 10 -9 15 -7c4 1 9 2 12 4c10 7 19 15 23 26c10 27 2 33 -7 46
c10 -6 24 -22 22 -39zM179 373c-6 -2 -12 -3 -16 2c-2 1 -1 2 1 3c7 6 15 10 19 18c2 -1 4 -1 4 -3c0 -7 -1 -16 -8 -20zM234 348c1 -2 -5 -5 -6 -6c-2 3 -8 4 -7 9s10 7 13 -3zM292 375c0 -10 0 -14 2 -23c5 -20 40 -75 60 -98c40 -47 88 -85 148 -97c-29 5 -44 3 -50 3
c-13 -1 -23 -2 -35 -5c-29 -6 -55 -13 -81 -27c2 -15 1 -34 -2 -41c-4 -10 -15 -22 -26 -31c-30 -23 -66 -12 -96 2c-7 -11 -14 -21 -22 -31c-10 4 -21 17 -25 18c-8 2 -29 -1 -29 -1s4 41 7 62c1 5 4 6 5 8c8 3 13 -3 18 -9c9 -15 16 -29 20 -46c8 -2 3 7 3 9
c-6 18 -11 41 -26 53c-8 6 -21 6 -25 -2c-3 -4 -1 -9 -6 -10c-3 0 -4 0 -6 2c-17 14 -29 30 -36 45c-2 4 6 11 7 14c-7 2 -8 -6 -23 -9c-6 -1 -11 -2 -17 0c-7 2 -12 5 -17 9c-4 6 -6 11 -2 17c9 9 23 11 41 14c-15 5 -29 9 -41 30c-9 16 -27 67 1 81c5 2 17 -9 24 -12
c6 -3 8 0 7 2c-16 8 -35 26 -41 38c-3 7 2 14 5 16c15 6 36 -14 50 -25c-9 19 8 27 21 28c39 2 80 8 110 -21c8 -9 15 -22 12 -34c-3 -14 -23 -32 -23 -32s3 -5 4 -4c22 13 68 58 84 107zM534 225c7 -19 12 -43 2 -57c-7 -9 -39 -6 -58 2c-88 36 -157 138 -172 188
c-9 26 0 50 23 51c49 3 119 -61 144 -88c26 -29 52 -67 61 -96zM468 223c-19 35 -44 58 -68 87c-45 55 -55 42 -62 72c-4 -36 35 -84 63 -118c26 -32 66 -71 98 -76c-1 14 -20 7 -21 18c-2 6 -6 9 -10 17zM318 95c-4 12 -16 7 -16 7s-4 19 -10 14c-1 -2 2 -4 3 -7
c1 -5 2 -10 -2 -7c-3 2 -7 8 -4 15c0 1 5 3 2 5c-2 2 -6 -1 -7 -4c-4 -11 5 -25 14 -22c-2 -6 -2 -16 -7 -21c-12 -12 -29 29 -34 14c10 -10 21 -27 36 -24c19 3 4 28 14 35c3 3 6 -10 11 -5zM320 106c4 -2 5 2 5 3c2 7 -5 17 -11 14c-2 -3 1 -8 2 -11c0 -4 3 -4 4 -6z
M322 114c0 -3 0 -4 -1 -4s-2 0 -2 2c-1 3 -2 4 -2 5c1 2 1 4 3 2c1 -2 2 -2 2 -5z" />
</svg>
```

Reply:
Yes. Since it is more convoluted to extract it from the icon font, I am suggesting you use the one that is already in SVG form: https://www.snort.org/assets/logo_snort_color.svg.

Reply:
I was able to remove all the extra stuff in Boxy SVG by repeatedly using **Object** > **Ungroup**, deleting extra elements, and using **Shape** > **Unite** to merge separate paths together:

![Screenshot 2024-06-05 at 17 40 19 (Arc)](https://github.com/simple-icons/simple-icons/assets/47499684/d3448b30-db4f-4fe7-b3cd-f75daa18e3be)

Reply:
I am not able to make it as good as you, but that looks great!

Reply:
What issue are you running into?

Reply:
It is ok, I managed to do the same thing using Illustrator
Here is the preview version
![image](https://github.com/simple-icons/simple-icons/assets/46492476/3feae9d7-6ed5-42be-a242-e9b8854ad35d)

Reply:
(I am able to remove the round R to if needed)

Reply:
Commit it!

Reply:
Nice!
Do you prefer with or without the round R?